internal/pkg/cli/override.go

```diff
@@ -20,13 +20,15 @@ import (
 const (
 	// IaC options for overrides.
 	cdkIaCTool = "cdk"
+	yamlPatch  = "CloudFormation YAML patches"
```

Comment:
Can we do:
```suggestion
	yamlPatch  = "yamlpatch"
```

And update the help menu here:
https://github.com/aws/copilot-cli/blob/318be13f9280352de52983470abd23cd3ccf9039/internal/pkg/cli/override.go#L148-L151

with an explanation of `yamlpatch`, for example:
```go
	help := `The AWS Cloud Development Kit (CDK) lets you override templates using
the expressive power of programming languages.
This option is recommended for users that need to override several resources.
To learn more about the CDK: https://aws.github.io/copilot-cli/docs/developing/overrides/cdk/

CloudFormation YAML patches is recommended for users that need to override
a handful resources or do not want to depend on any other tool.
To learn more about CFN yaml patches: https://aws.github.io/copilot-cli/docs/developing/overrides/yamlpatch/
`
```

Reply:
![](https://media.tenor.com/s1w62ls6xlQAAAAC/princess-bride.gif)